ui/components/multichain/pages/connections/components/site-not-connected.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { SiteNotConnected } from './site-not-connected';
+
+export default {
+  title: 'Components/Multichain/SiteNotConnected',
```

Comment:
This should follow the folder path convention
```suggestion
  title: 'Components/Multichain/Pages/Connections/Components/SiteNotConnected',
```

Reply:
Good catch, I thought the directory structure for storybook was flat